dockorm/container.py

```diff
@@ -103,8 +103,9 @@ def format_links(self):
         return {}
 
     volumes_readwrite = Dict()
-
     volumes_readonly = Dict()
+    volumes_no_bind = []
```

Comment:
I changed the variable to be defined as a traitlet, is there anywhere else I need to change? This value is actually being set here: https://github.com/quantopian/research_env/pull/108/files#diff-cd2871a3c46fc48002b2ad447d09eb4fR201 but I'm not sure if this is what you intended (as I'm not exactly sure how the traitlets work)